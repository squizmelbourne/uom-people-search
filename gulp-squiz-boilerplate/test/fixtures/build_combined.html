<!-- build:sass test/.tmp/styles/combined.css -->
    <!-- build:css test/.tmp/styles/combined.scss -->
        <!-- import:css test/fixtures/styles/combined*.scss -->
    <!-- endbuild -->
<!-- endbuild -->
<!-- build:sass test/.tmp/styles/combined_two.css -->
    <!-- build:css test/.tmp/styles/combined_two.scss { "media": "print"} -->
        <!-- import:css test/fixtures/styles/other.css -->
    <!-- endbuild -->
<!-- endbuild -->