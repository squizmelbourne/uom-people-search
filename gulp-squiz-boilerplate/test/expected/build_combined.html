<link href="test/.tmp/styles/combined.css" rel="stylesheet" />
<link href="test/.tmp/styles/combined_two.css" rel="stylesheet" />